calicoctl/commands/node/run.go

```diff
@@ -205,6 +203,21 @@ Description:
 		// the AS number, so convert.
 		asNumber = argutils.ValidateASNumber(asNumber).String()
 	}
+	// CLI flags will take precedence over env vars
```

Comment:
Rob, I can get those added in, I'm going to run through the reference list again as well to see if I missed anything.